src/fsharp/FSharp.Core/list.fs

```diff
@@ -223,7 +223,7 @@ namespace Microsoft.FSharp.Collections
 
         [<CompiledName("Pairwise")>]
         let pairwise (list: 'T list) =
-            Microsoft.FSharp.Primitives.Basics.List.pairwise list
```

Comment:
I think that's my fault. I also did such cleanups and IIRC they are merged already.